### ISSUE TITLE
Can't use fhirpath cli (0.10.2)

### ISSUE THREAD

**Issue by col-panic** [lhncbc/fhirpath.js#79]
Executing fhirpath cli leaves me with

```
marco@marcos-mbp ~> fhirpath
internal/modules/cjs/loader.js:615
    throw err;
    ^

Error: Cannot find module 'js-yaml'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:613:15)
    at Function.Module._load (internal/modules/cjs/loader.js:539:25)
    at Module.require (internal/modules/cjs/loader.js:667:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/fhirpath/bin/fhirpath:5:14)
    at Module._compile (internal/modules/cjs/loader.js:738:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:749:10)
    at Module.load (internal/modules/cjs/loader.js:630:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:570:12)
```

after installing it using `npm install -g fhirpath`

**Comment by plynchnlm** [comment]
Thanks for the report.  I've confirmed it.  The cause is that js-yaml is listed as only a development dependency.  I will try to get fix out soon.

**Comment by plynchnlm** [comment]
Fixed in 0.10.3.
